.github/workflows/action-install-from-conda.yml

```diff
@@ -0,0 +1,108 @@
+# Install esmvalcore from PyPi on different OS's
+# and different Python version; test locally with
+# act: https://github.com/nektos/act
+# Example how to setup conda workflows:
+# https://github.com/marketplace/actions/setup-miniconda
+# Notes:
+#  - you can group commands with | delimiter (or &&) but those will be run
+#    in one single call; declaring the shell variable makes the action run each
+#    command separately (better for debugging);
+#  - can try multiple shells eg pwsh or cmd /C CALL {0} (but overkill for now!);
+# FIXME OSX/Python3.8: 3.8.5 available on OSX only from July 20, 2020
+# TODO: read the cron tasking documentation:
+# https://www.netiq.com/documentation/cloud-manager-2-5/ncm-reference/data/bexyssf.html
+
+name: Conda Base Install
+
+# runs on a push on master and at the end of every day
+on:
+  # triggering on push without branch name will run tests everytime
+  # there is a push on any branch
+  # turn it on only if needed
+  push:
+    branches:
+    - master
+    - github-actions2
```

Comment:
This can probably be removed, I guess

Reply:
I actually want to keep it and the branch up as well - saves us time in case we need to make any tweaks in the future